centos/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM centos:8
+FROM centos:7
```

Comment:
I missed a slack thread where it was decided to not do this downgrade -- reverting